packages/ddp/livedata_tests.js

```diff
@@ -601,29 +611,109 @@ if (Meteor.isClient) {
       }
     ];})());
 
-    testAsyncMulti("livedata - publish multiple cursors", [
+  testAsyncMulti("livedata - publisher errors with onStopped callback", (function () {
```

Comment:
I guess I could do a for loop and define some function like "onStoppedPretendingToBeOnError" but this seemed easier to understand.